test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -1,26 +1,25 @@
 'use strict';
 /*
  * This test is a regression test for joyent/node#8900.
+ *
+ * Node.js 0.10.34 contains the bug that this test detects.
+ * Node.js 0.10.35 contains the fix.
+ *
+ * Due to the introduction of ES6-isms, comment out `'use strict'` and
+ * `require('../common');` to run the test on Node.js 0.10.x.
  */
-const common = require('../common');
+require('../common');
 
-const TEST_DURATION = common.platformTimeout(100);
 const N = 5;
 var nbIntervalFired = 0;
 
-const keepOpen = setTimeout(() => {
-  console.error('[FAIL] Interval fired %d/%d times.', nbIntervalFired, N);
-  throw new Error('Test timed out. keepOpen was not canceled.');
-}, TEST_DURATION);
+const keepOpen = setInterval(function() {}, 9999);
```

Comment:
It's less arbitrary because it is a `setInterval()` and not a `setTimeout()`. It could just as well be a `net.createServer()` or anything else that will keep the test running forever if not canceled.